cargo-miri/bin.rs

```diff
@@ -1173,8 +1181,22 @@ fn main() {
 
     match args.next().as_deref() {
         Some("miri") => phase_cargo_miri(args),
-        Some("rustc") => phase_rustc(args, RustcPhase::Build),
+        // Work around a cargo bug where it ignores `RUSTC_WRAPPER`, so we get
+        // called ourselves and handle the special case for `rustc -vV`.
+        Some("-vV") => {
```

Comment:
When I tried this, `cargo miri test` in our test-cargo-miri failed because the libc build script invokes `RUSTC --version`...

Reply:
huh... it works in the ui test suite, I didn't check test-cargo-miri

Reply:
Yeah it fails that way on CI now

> error: failed to run custom build command for `libc v0.2.126`